website/screens/components/number-input/specs/NumberInputSpecsPage.tsx

```diff
@@ -0,0 +1,108 @@
+import Image from "next/image";
+import numberSpecs from "./images/number_specs.png";
+import numberInputSpecs from "./images/number_input_states.png";
+import numberInputStates from "./images/number_action_states.png";
+import numberAnatomy from "./images/number_anatomy.png";
+import HeadingLink from "../../../common/HeadingLink";
+import {
+  DxcStack,
+  DxcText,
+  DxcLink,
+  DxcList,
+} from "@dxc-technology/halstack-react";
+import Figure from "../../../common/Figure";
+import Code from "../../../common/Code";
+import DocFooter from "../../../common/DocFooter";
+
+const NumberInputSpecs = () => {
+  return (
+    <DxcStack gutter="xxxlarge">
+      <DxcStack gutter="large">
+        <HeadingLink level={2}>Specifications</HeadingLink>
+        <Figure caption="Design specifications of the number input component">
+          <Image
+            src={numberSpecs}
+            alt="Design specifications of the number input component"
+          />
+        </Figure>
+        <DxcText as="p">
+          The number input <Code>color</Code>, <Code>typography</Code>,{" "}
+          <Code>border</Code>, <Code>spacing</Code>, <Code>width</Code> and{" "}
+          <Code>margin</Code> specifications are inherited from the text input,
+          for reference{" "}
+          <DxcLink
+            href="https://developer.dxc.com/design/guidelines/components/text-input"
```

Comment:
I think this should be a link to `/components/text-input/specifications`.

Reply:
Done.